.circleci/config.yml

```diff
@@ -102,7 +102,7 @@ jobs:
               echo $PLOTLY_GA_CODE > ~/dash-sample-apps/apps/"$APP"/assets/plotly_ga.js
               git add ~/dash-sample-apps/apps/"$APP"/assets/plotly_ga.js && git commit -m "Deployed commit: $CIRCLE_SHA1"
               git remote add $APP https://dash-gallery.plotly.host/GIT/$APP
-              git push $APP master --force
+              git push $APP master --force || echo "Deploy failed because there is no such app on Dash Gallery: $APP. Please use the web interface to create an app with this name."; exit 1
```

Comment:
I'm not too familiar with bash script. `Exit 1` will end the loop, is it possible to continue the loop so the other apps will be able to deploy?

Reply:
Good point! Fixed in f89768f.